src/main/java/com/wootecobook/turkey/comment/controller/api/CommentApiController.java

```diff
@@ -0,0 +1,74 @@
+package com.wootecobook.turkey.comment.controller.api;
+
+import com.wootecobook.turkey.comment.service.CommentService;
+import com.wootecobook.turkey.comment.service.dto.CommentCreate;
+import com.wootecobook.turkey.comment.service.dto.CommentResponse;
+import com.wootecobook.turkey.comment.service.dto.CommentUpdate;
+import com.wootecobook.turkey.commons.resolver.UserSession;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.web.PageableDefault;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+
+import static org.springframework.hateoas.mvc.ControllerLinkBuilder.linkTo;
+
+@RestController
+@RequestMapping("/api/posts/{postId}/comments")
+public class CommentApiController {
+
+    private final CommentService commentService;
+
+    public CommentApiController(final CommentService commentService) {
+        this.commentService = commentService;
+    }
+
+    @GetMapping
+    public ResponseEntity<Page<CommentResponse>> list(@PageableDefault(size = 5, sort = "createdAt") Pageable pageable,
+                                                      @PathVariable Long postId) {
+
+        final Page<CommentResponse> commentResponses = commentService.findCommentResponsesByPostId(postId, pageable);
+        return ResponseEntity.ok(commentResponses);
+    }
+
+    //TODO /{id}는 comment 하나만 조회인데 자식List 조회해도 괜찮은지?
```

Comment:
list와 같은 기능이라면 아래 childCommentList 보다는 list 핸들러메서드에서 한번에 처리하는게 좋을 것 같아요. 페이지를 사용하고 있으니깐 쿼리스트링으로 page를 다르게 받아서 페이지를 처리하는 형식으로요.
```suggestion
   /api/posts/{postId}/comments?page=0 // 0번 페이지 요청 (즉, 첫 5개 댓글 요청)
   /api/posts/{postId}/comments?page=1 // 1번 페이지 요청 (즉, 두번째 5개 댓글 요청)
```
같은 형식으로 생각해봤어요 ㅎㅎ 다른분들 의견도 한번 들어보면 좋을 것 같아요

Reply:
childCommentList 는 답글 전용 조회입니다. 
처음에 글 보면 댓글만 조회되고
해당 댓글에서 답글 보기 누르면 답글을 조회해서 보여주려고 이렇게 했습니다.
문제는 uri 컨벤션에 안 맞아서 나은 방법이 있을까 질문을 남겼습니다.